vhdbuilder/packer/install-dependencies.sh

```diff
@@ -236,6 +236,7 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+1.2.5
```

Comment:
Please also update linux-vhd-content-test.sh file

Reply:
I do not find this file, can you provide me the path to it ?

Reply:
https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/test/linux-vhd-content-test.sh

Reply:
Looks like this is a new file, my repository does not have this file, i will update this file in a separate PR, do not want to mess with rebase and etc, if thats okay

Reply:
don't think anything will be messed up if you rebase,  but if you prefer not to do so, we can have two PRs approved and merge one after the other so that the tests don't fail.